packages/app/src/systems/Vault/services/VaultClient.ts

```diff
@@ -28,6 +33,14 @@ export class VaultClient extends EventEmitter {
     VaultServer.methods.forEach((methodName) => {
       this[methodName] = this.createRequest(methodName);
     });
+
+    if (IS_CRX) {
+      chrome.runtime.onMessage.addListener((e: LockChangeEvent) => {
```

Comment:
With the implementation on the server of notify you need to implement [here](https://github.com/FuelLabs/fuels-wallet/blob/0248a70099d9202043a825da917ed2a4673311aa/packages/app/src/systems/Vault/connectors/VaultCRXConnector.ts#L27) and here to receive the correct type.